### PR TITLE
[RTM] Hotfix add support default values

### PR DESCRIPTION
**Pull request by zonky2**
Hotfix add support default values

https://github.com/MetaModels/contao-frontend-editing/issues/24

### CODE REVIEW

File: src/View/ActionHandler/CreateHandler.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 /**
  * This file is part of contao-community-alliance/dc-general-contao-frontend.
  *
- * (c) 2015 Contao Community Alliance.
+ * (c) 2017 Contao Community Alliance.
```
Comment (richardhj) [inline src/View/ActionHandler/CreateHandler.php]: Regularly, we keep the "2015"

File: src/View/ActionHandler/CreateHandler.php [hunk 2]
```diff
@@ -51,10 +52,23 @@ public function process()
             return;
         }
 
-        $dataProvider = $environment->getDataProvider();
```
Comment (richardhj) [inline src/View/ActionHandler/CreateHandler.php:54]: Please change  to `@return void` in the PHPdoc of the method

File: src/View/ActionHandler/CreateHandler.php (hunk 1 above)
Comment (zonky2) [reply src/View/ActionHandler/CreateHandler.php]: may be 
```
 * (c) 2015 - 2017 Contao Community Alliance.
```
like https://github.com/MetaModels/core/issues/1150

**Review by zonky2 — COMMENTED** [review]
o.k.

**Review by richardhj — APPROVED** [review]

File: src/View/ActionHandler/CreateHandler.php (hunk 1 above)
Comment (discordier) [reply src/View/ActionHandler/CreateHandler.php]: Should become a time span updating the last section when file get's touched AFAIR.

Comment (zonky2) [reply src/View/ActionHandler/CreateHandler.php]: noch anders?
https://github.com/contao-community-alliance/dc-general-contao-frontend/pull/8/files#diff-3a0131e0a7119bc1eeccbb0cbdecb44cR6

**Pull request merged**